server/src/internalClusterTest/java/org/elasticsearch/plugins/IndexFoldersDeletionListenerIT.java

```diff
@@ -132,7 +132,9 @@ public void testListenersInvokedWhenIndexIsDeleted() throws Exception {
         reason = "Debug #93226",
         value = "org.elasticsearch.indices.cluster.IndicesClusterStateService:DEBUG,"
             + "org.elasticsearch.indices.IndicesService:DEBUG,"
-            + "org.elasticsearch.index.IndexService:DEBUG"
+            + "org.elasticsearch.index.IndexService:DEBUG,"
+            + "org.elasticsearch.env.NodeEnvironment:DEBUG,"
+            + "org.elasticsearch.cluster.service.MasterService:TRACE"
```

Comment:
I'm curious about how `MasterService` would help debugging this issue, do you think that applying the cluster state is somehow related to the issue? (other than closing the index)